src/classes/CRLP_RollupProcessingOptions.cls

```diff
@@ -155,7 +155,7 @@ public class CRLP_RollupProcessingOptions {
          * Used to expose this value to the LDV Batch Job to retrieve the current setting.
          */
         public Map<Id,Set<Id>> getUpdatedMapOfProcessedOppsByAccountId() {
```

Comment:
This method name can also be refactored since it's no longer retrieving the map "ByAccountId". Sorry if this was missed in the WI.